src/Shared/XMakeAttributes.cs

```diff
@@ -40,9 +40,11 @@ internal static class XMakeAttributes
         internal const string taskName = "TaskName";
         internal const string continueOnError = "ContinueOnError";
         internal const string project = "Project";
+        internal const string @implicit = "_Implicit";
```

Comment:
Can this be isImplicit so you can get rid of the `@`?  Should the attribute name just be `IsImplicit`?